### PR TITLE
Add 2020-04-14 incubator call notes

### PR DESCRIPTION
**Pull request by syg**

### CODE REVIEW

File: incubator-calls/2020-04/april-14.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+DE: I think these inline module fallbacks are interesting to study. Worth looking into other kinds of fallbacks. I don’t think things that were claimed in that thread were concrete [??]
+
+RG: I don’t think they were concrete.
+
+DE: I’m saying the fallbacks are orthogonal. There are other examples, script tags have version fallbacks. There’d be all kinds of feature testing (e.g. are async generators there) that’d be good to do but they would be separate things. That case is like different module attributes.
+
+RG: That’s the hazard. If the graph itself fails, then you have to ship different source.
+
+JHD: If I have an unknown tag in an image tag, does the HTML page break?
```
Comment (ljharb) [inline incubator-calls/2020-04/april-14.md:95]:
```suggestion
JHD: If I have an unknown image type in an image tag, does the HTML page break?
```

**Comment by littledan** [comment]
Do you want to put these here, or in the incubator agendas repository? I've been putting notes for champion group and outreach group calls together with them in their own repositories.

**Comment by syg** [comment]
> Do you want to put these here, or in the incubator agendas repository? I've been putting notes for champion group and outreach group calls together with them in their own repositories.

Either works. Since the agenda repos are already separate, guess that suggestion makes more sense.

**Review by ljharb — COMMENTED** [review]
forgot to submit this a few days ago

**Pull request closed without merging**